kano_settings/boot_config.py

```diff
@@ -241,8 +244,7 @@ def __init__(self, path):
         self.base = os.path.basename(path)
         self.dir = os.path.dirname(path)
 
-        self.lockpath = os.path.join(lock_dir,
-                                     'kano_config_'+self.base+'.lock')
+        self.lockpath = os.path.join(lock_dir, 'kano_config_' + self.base + '.lock')
```

Comment:
If you're fixing this then you might as well switch to use the `"kano_config_{}.lock".format()` style.